scripts/automatic_endpoint_update.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+SHELL=/bin/sh
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+
+chmod +rx query-endpoint-resources.sh; ./query-endpoint-resources.sh
+chmod +rx populatedb_prod.sh; ./populatedb_prod.sh
```

Comment:
I'm not convinced that updating the endpoints should be done at the same time as updating NPPES. I think we'd want to update the endpoints weekly, but I think NPPES updates monthly. If updating the NPPES data wasn't such a hassle i'd say leave it as is, but it's a multi-hour process. I think having this script here is fine for running monthly, but it would be nice to have a script that could run more frequently that just pulls the endpoint lists and adds them to the database. What do you think?